### ISSUE TITLE
Submit item with null value

### ISSUE THREAD

**Issue by michele-pauluzzi** [bostrowski/Pretius-APEX-Enhanced-Lov-Item#27]
When you submit an item without a value selected, you get this errror
![image](https://user-images.githubusercontent.com/54415530/69859759-129e1b00-1295-11ea-9d99-14d97cfe74df.png)

The error is at line 1995 enhancedLovItem.js
if pValue is null you get the error above.

```
`  /*
    *
    * function name: _elementSetValue
    * description  : 
    * params:
    *   -
    *   -
    *
  */  
  _elementSetValue: function( pValue ){
    apex.debug.message(this.C_LOG_DEBUG, this.logPrefix, '_elementSetValue', {
      "arguments": arguments,
      "pValue": pValue
    });

    this.element.val( pValue );
  },`
```

**Comment by bostrowski** [comment]
@michele-pauluzzi 
Yes, you are playing with not yet released v1.1.0. Btw. Please send me e-mail to ostrowski.bartosz@gmail.com

**Comment by michele-pauluzzi** [comment]
Fixed
